samples/tab-stage-view/csharp/TabInStageView/README.md

```diff
@@ -0,0 +1,69 @@
+﻿# TabInStageView
+
```

Comment:
Let's add a link to document here and also provide some info about the sample here

Reply:
Added link